rosbag_filepath: "/home/victor/data/building_sim/terrain_coverage_2019-01-21-23-26-31.bag"
skip_first_n_sec: 50
ground_truth_robot_pose_frame: "robot_ground_truth"
odom_simulator_input_topic: "/firefly/ground_truth/odometry"
ground_truth_tsdf_layer_filepath: "/home/victor/data/voxgraph/ground_truth_maps/burning_building_rubble"
registration_methods: ["explicit_to_implicit", "implicit_to_implicit"]
sampling_ratios: [-1,
                  1.0,    0.9,    0.8,    0.7,    0.6,    0.5,    0.4,    0.3,    0.2,    0.1,
                  0.10,   0.09,   0.08,   0.07,   0.06,   0.05,   0.04,   0.03,   0.02,   0.01,
                  0.010,  0.009,  0.008,  0.007,  0.006,  0.005,  0.004,  0.003,  0.002,  0.001,
                  0.0010, 0.0009, 0.0008, 0.0007, 0.0006, 0.0005, 0.0004, 0.0003, 0.0002, 0.0001, 0.0000]
